src/index.html

```diff
@@ -6,9 +6,36 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="Logo">
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link">
              Samsung
            </a>
          </li>
```
you should format all of them like this, not only one